scripts/docs_version_pragma_check.sh

```diff
@@ -119,8 +119,13 @@ function findMinimalVersion()
 
     if [ -z "$version" ]
     then
-        printError "No release $sign$pragmaVersion was listed in available releases!"
-        exit 1
+        if [[ "$greater" = true && "$pragmaVersion" =~ 99 ]]
```

Comment:
Does this mean that if version string is zero, instead of exiting with `1`, we check if `greater` is true and `pragmaVersion`  equals the regex expression `99`, so something like `>99`? Could you briefly explain why this is needed?

Reply:
For code snippets that require 0.8.0, we cannot write `>= 0.8.0` because that version does not exist. Instead, we write `>0.7.99`, which is correct before and after the release.

It is just a quick fix. After the 0.8 release, we should change the version pragmas in the documentation and this fix should maybe also be improved at some point.